app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: callable) -> callable:
+    results = {}
+
+    def wrapper(*args, **kwargs) -> callable:
```

Comment:
There is no need to process `**kwargs` in this decorator, limit processing to just `*args`